### ISSUE TITLE
Type Error on Labels 

### ISSUE THREAD

**Issue by jacobdaviescam** [haonan-li/TWM-metonymy-resolution#2]
I am attempting to replicate some results from your model and am coming into a TypeError in the line:
utils.metonymy.py, line 147, in load_and_cache_examples
all_labels = torch.tensor([f.label for f in features], dtype=torch.long)
![Screenshot_20230109_212822](https://user-images.githubusercontent.com/119947345/211412087-83016d3d-01b0-4410-bb07-bfd1a6eda2b1.png)

**Comment by haonan-li** [comment]
This bug has been fixed. 
You can remove the cached file, and rerun the code.
